File: tests/fixtures/lang_samples/go/conditionals.go
package conditionals

func ThreeOps(a, b, c, d bool) int {
	if a && b && c && d {
		return 1
	}
	return 0
}

func SixOps(a, b, c, d, e, f, g bool) int {
	for a && b && c && d && e && f && g {
		return 1
	}
	return 0
}

func TwoOps(a, b, c bool) int {
	if a && b || c {
		return 1
	}
	return 0
}

File: tests/fixtures/lang_samples/java/Conditionals.java
public class Conditionals {
    public int threeOps(boolean a, boolean b, boolean c, boolean d) {
        if (a && b && c && d) {
            return 1;
        }
        return 0;
    }

    public int sixOps(boolean a, boolean b, boolean c, boolean d, boolean e, boolean f, boolean g) {
        while (a && b && c && d && e && f && g) {
            return 1;
        }
        return 0;
    }

    public int twoOps(boolean a, boolean b, boolean c) {
        if (a && b || c) {
            return 1;
        }
        return 0;
    }
}
